Formula/kubernetes-helm.rb

```diff
@@ -45,5 +48,6 @@ def install
 
     version_output = shell_output("#{bin}/helm version --client 2>&1")
     assert_match "GitTreeState:\"clean\"", version_output
+    assert_match stable.instance_variable_get(:@resource).instance_variable_get(:@specs)[:revision], version_output if build.stable?
```

Comment:
Actually, I just tested this locally and `if build.stable?` seems to do what's expected:
- If I install stable and test, it passes.
- If I install stable, modify the revision in the formula, and then test, it fails with expected output.
- If I install HEAD and test, it passes.
- If I install HEAD, but remove `if build.stable?` and test, it fails with expected output.